config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'welcome/homepage'
   devise_for :users
   resources :repairs
+  root 'welcome#homepage'
```

Comment:
Your `root` path should always be at the top of your `routes.rb` file since Rails looks for routes in order, from top to bottom.